demosplan/DemosPlanCoreBundle/Repository/FaqCategoryRepository.php

```diff
@@ -65,6 +65,28 @@ public function getFaqCategoriesByCustomer(Customer $customer): array
         }
     }
 
+    /**
+     * Get all static platformFaqCategories - same for all customers.
+     *
+     * @return PlatformFaqCategory[]
+     */
+    public function getCustomerIndependentPlatformFaqCategories(): array
+    {
+        try {
+            $query = $this->getEntityManager()
+                ->createQueryBuilder()
+                ->select('platformFaqCategory')
+                ->from(PlatformFaqCategory::class, 'platformFaqCategory')
+                ->orderBy('platformFaqCategory.title');
+
+            return $query->getQuery()->getResult();
+        } catch (NoResultException $e) {
+            $this->logger->error('Get platformFaqCategories failed.', [$e]);
+
+            return [];
+        }
```

Comment:
Minor:
This looks like the old verbose way.
Instead you can use something like this:
```suggestion
        $list = $this->findBy([], ['title' => 'ASC']);
        if (null === $list) {
            $this->logger->info('Get platformFaqCategories failed.');
            throw new EntityNotFoundException('Nothing found');
        }

        return $list;
```
I'm not sure how to handle the exception. Maybe we should bubble them up.